ldaptor/test/test_ldapclient.py

```diff
@@ -229,6 +233,45 @@ def test_send_multiResponse_ex(self):
         client.dataReceived(resp_bytestring)
         self.assertEqual((response.value, response.controls), self.successResultOf(d))
 
+    def test_send_multiResponse_ex_with_handler(self):
```

Comment:
There was no test covering `send_multiResponse_ex` when that method was passed a `handler`.